content/applications/sales/sales/products_prices/returns.rst

```diff
@@ -0,0 +1,101 @@
+Process returns and refunds
+===========================
+
+The :guilabel:`Sales` app provides two different ways to process returns based on whether an
+invoice has been sent or not.
+
+Before invoicing: Reverse Transfers
+===================================
+
+Returns are completed using *Reverse Transfers* when a customer decides to return a product before
+an invoice has been sent or validated.
+
+.. note::
+   In order to use Reverse Transfers, the :guilabel:`Inventory` app must also be installed.
+
+To start a return, navigate to the customer's sales order, and click on the :guilabel:`Delivery`
+smart button to open the associated delivery order.
+
+On the validated delivery order, click :guilabel:`Return` to open the :guilabel:`Reverse Transfer`
+pop-up window. By default, the :guilabel:`Quantity` matches the validated quantities from the
+delivery order. Update the quantities if necessary. Click on the trash icon next to a line item to
+remove it from the return.
+
+.. image:: returns/reverse-transfer-popup.png
+   :align: center
+   :alt: Reverse Transfer Pop-Up
+
+Click :guilabel:`Return` to confirm the return. This generates a new warehouse operation for the
+incoming returned product(s). Upon receiving the return, the warehouse team *validates* the
+warehouse operation. Then, on the original sales order, the :guilabel:`Delivered` quantity will
+reflect the difference between the initial validated quantities and the returned quantities.
+
+.. image:: returns/updated-sales-quantities.png
+   :align: center
+   :alt: Updated Sales Order Quantities
+
+When an invoice is created, the customer receives an invoice only for the products they are
+keeping.
+
+After invoicing
+===============
+
+Sometimes, customers return an item after they receive and/or pay for their invoice. In these
+cases, a return using only Reverse Transfers is impossible since validated or sent invoices cannot
+be changed. But Reverse Transfers can be used in conjunction with *Credit Notes* to complete the
+customer's return.
+
+To start a return, navigate to the relevant sales order. If there is a payment registered on the
+sales order, then the payment details will appear in the Chatter, and the invoice (accessible
+through the :guilabel:`Invoices` smart button) will have a green banner across it.
+
+From the sales order, click on the :guilabel:`Delivery` smart button to view the validated delivery
+order. Then click :guilabel:`Return` to open the :guilabel:`Reverse Transfer` pop-up window. Edit
+the :guilabel:`Product` or :guilabel:`Quantity` as needed for the return, then click
+:guilabel:`Return`. This generates a new warehouse operation for the incoming returned product(s),
+which is validated by the warehouse team once the return is received. Then, on the sales order, the
+:guilabel:`Delivered` quantity will reflect the difference between the initial validated quantities
+and the returned quantities.
+
+.. image:: returns/case-2-updated-sales-quantities.png
```

Comment:
@meng-odoo This image should be retaken to be a little more zoomed in, and the "Delivered" column should be highlighted with a red box around it.

Make sure the product, quantity, and customer match the other screenshots in this doc for continuity!

Reply:
@mivu-odoo @samueljlieber Updated image attached
![case-2-updated-sales-quantities](https://user-images.githubusercontent.com/101904966/188997374-d4f348df-6397-4341-80e3-bd60aadb1bce.png)